packages/wrangler/src/publish/list.ts

```diff
@@ -0,0 +1,35 @@
+import { fetchResult } from "../cfetch";
+import { logger } from "../logger";
+
+export type ServicesListRes = [
+	{
+		id: string;
+		created_on: string;
+		default_environment: {
+			script: {
+				last_deployed_from: string;
+			};
+		};
+	}
+];
+
+export async function list(
+	accountId: string | undefined,
+	name: string | undefined
+) {
+	let url = `/accounts/${accountId}/workers/services`;
+	if (name) {
+		url += `?name=${name}`;
```

Comment:
Since you are already using template literals, you can keep the syntax consistent here: 
```suggestion
`${url}?name=${name}`;
```